### PR TITLE
Complete whiteboard.

### PR DESCRIPTION
**Pull request by nathanrhead**

### CODE REVIEW

**Comment by nathanrhead** [comment]
- [x] Top-level README “Table of Contents” is updated
 - [] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

File: javascript/code-challenges/data-structures/queueWithStacks/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+## API: A description of each method publicly available to your queue.
+
+- push: takes any value as an argument and adds a new node with that value to the top of the stack--but (somehow) to the end of the queue--with an O(1) time performance.
+
+- pop: takes no argument; removes the node from the top of the stack--the front of the queue--and returns the node’s value, raising an exception on an empty queue.
+
+- peek: takes no argument; returns the value of the node located on top of the stack--or the front of the queue--without removing it from the stack, raising an exception on an empty queue.
+
+## Resources and Links
```
Comment (adrianhuebner) [inline javascript/code-challenges/data-structures/queueWithStacks/README.md:31]: Everything looks really great Nathan, would just like to see some tests added!
